F suggestions ts (#7)
krlmlr/tidyprog: script/73-enquos.Rmd

```
</details>

There are cases though, when you need knowledge about what the user added to the ellipsis.

> This is then handled by capturing the content in a quosure, which can be unquoted by the `!!!`-operator.

Reviewer: ```suggestion
This is then handled by capturing the content in a list of quosures, which can be unquoted by the `!!!`-operator.
```

> You need the "triple-bang" operator here, because the ellipsis can hold more than one variable.

Reviewer: ```suggestion
You need the "triple-bang" operator here, because the ellipsis can hold more than one expression.
```

}
```

> We didn't need to capture the content, but it still works, as we can see here:

Reviewer: I don't understand "need to capture", perhaps:

```suggestion
We didn't need to process the content, but it still works, as we can see here:
```
